src/components/Modal/Modal.vue

```diff
@@ -26,15 +26,14 @@
 <template>
 	<div>
 		<button @click="showModal">Show Modal</button>
-		<modal v-if="modal" @close="closeModal">
+		<modal v-if="modal" @close="closeModal" size="prompt">
 			<div class="modal__content">Hello world</div>
 		</modal>
 	</div>
 </template>
 <style scoped>
 	.modal__content {
-		width: 50vw;
-		margin: 10vw 0;
+		margin: 50px;
```

Comment:
defining a width is no longer necessary as the modal has a distinct size out of the box